eventbus/jetstream/sensor/trigger_conn.go

```diff
@@ -227,6 +237,14 @@ func (conn *JetstreamTriggerConn) processMsg(
 		return
 	}
 
+	// De-duplication
+	// In the off chance that we receive the same message twice, don't re-process
+	_, alreadyReceived := conn.recentMsgsByID[event.ID()]
```

Comment:
If same ID message comes every 2 mins, it will be blocked here, right?

Reply:
I believe we resolved this offline